<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>771</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Configuración Activa</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>561</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Formulario de Configuración App.</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_titulo_app">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>310</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>25</y>
           <width>130</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Título en Menú de la App:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>25</y>
           <width>50</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Logo App:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_logo_app">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>50</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>45</y>
           <width>140</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Título Principal en la Imagen:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_hero_titulo">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>45</y>
           <width>310</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_imagen_central">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>250</y>
           <width>104</width>
           <height>104</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>104</width>
           <height>104</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>104</width>
           <height>104</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-radius: 50px;
min-width: 100px;
min-height: 100px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="btnHeroImagen">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>220</y>
           <width>40</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_hero_imagen">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>220</y>
           <width>350</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>225</y>
           <width>80</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Imagen Portada:</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnAgregarConfig">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>370</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnModificarConfig">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>370</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Modificar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnDesactivarConfig">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>370</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Desactivar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnLimpiarFormulario">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>370</y>
           <width>80</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Limpiar</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="Tabla_configuracion_activa">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>400</y>
           <width>740</width>
           <height>150</height>
          </rect>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>id</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Titulo de Menú</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Logo App</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Icono Abrir</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Icono Cerrar</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Titulo Principal</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Imagen Central</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Texto Pie Página</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>75</y>
           <width>150</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Subtítulo y Texto pie de página:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_footer_texto">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>70</y>
           <width>310</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btnCerrar">
         <property name="geometry">
          <rect>
           <x>494</x>
           <y>10</y>
           <width>31</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border : none;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../assets/iconos/cerrar.png</normaloff>../assets/iconos/cerrar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="btnLogo">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>20</y>
           <width>70</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Buscar Logo</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_logo_app">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>80</y>
           <width>54</width>
           <height>54</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-radius: 25px;
min-width: 50px;
min-height: 50px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_icono_abrir">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>175</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btnIconoAbrir">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>145</y>
           <width>70</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Buscar Icono</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_icono_abrir">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>205</y>
           <width>54</width>
           <height>54</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-radius: 25px;
min-width: 50px;
min-height: 50px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>150</y>
           <width>90</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Icono Menú Abrir:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_icono_cerrar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>295</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_icono_cerrar">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>325</y>
           <width>54</width>
           <height>54</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>54</width>
           <height>54</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-radius: 25px;
min-width: 50px;
min-height: 50px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="btnIconoCerrar">
         <property name="geometry">
          <rect>
           <x>590</x>
           <y>265</y>
           <width>70</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Buscar Icono</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>490</x>
           <y>270</y>
           <width>100</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Icono Menú Cerrar:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>100</y>
           <width>100</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Dirección facebook:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_direccion_facebook">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>95</y>
           <width>310</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>125</y>
           <width>100</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Dirección instagram:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_direccion_instagram">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>120</y>
           <width>310</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>65</x>
           <y>150</y>
           <width>80</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Dirección twitter:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_direccion_twitter">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>145</y>
           <width>310</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>175</y>
           <width>90</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Dirección youtube:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_direccion_youtube">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>170</y>
           <width>310</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_direccion_correo">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>195</y>
           <width>310</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>200</y>
           <width>90</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Correo Electrónico:</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Configuración Inactivas</string>
    </attribute>
    <widget class="QTableWidget" name="Tabla_configuraciones_inactiva">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>741</width>
       <height>481</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>id Usuario</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>D.N.I</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Domicilio</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Localidad</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Provincia</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Usuario</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Contraseña</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ruta Foto</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Rol</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btnReactivarConfiguracion">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>510</y>
       <width>141</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Reactivar Configuración</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_titulo_app</tabstop>
  <tabstop>lineEdit_hero_titulo</tabstop>
  <tabstop>lineEdit_footer_texto</tabstop>
  <tabstop>lineEdit_direccion_facebook</tabstop>
  <tabstop>lineEdit_direccion_instagram</tabstop>
  <tabstop>lineEdit_direccion_twitter</tabstop>
  <tabstop>lineEdit_direccion_youtube</tabstop>
  <tabstop>lineEdit_direccion_correo</tabstop>
  <tabstop>btnHeroImagen</tabstop>
  <tabstop>lineEdit_hero_imagen</tabstop>
  <tabstop>btnLogo</tabstop>
  <tabstop>lineEdit_logo_app</tabstop>
  <tabstop>btnIconoAbrir</tabstop>
  <tabstop>lineEdit_icono_abrir</tabstop>
  <tabstop>btnIconoCerrar</tabstop>
  <tabstop>lineEdit_icono_cerrar</tabstop>
  <tabstop>btnAgregarConfig</tabstop>
  <tabstop>btnModificarConfig</tabstop>
  <tabstop>btnDesactivarConfig</tabstop>
  <tabstop>btnLimpiarFormulario</tabstop>
  <tabstop>Tabla_configuracion_activa</tabstop>
  <tabstop>btnCerrar</tabstop>
  <tabstop>Tabla_configuraciones_inactiva</tabstop>
  <tabstop>btnReactivarConfiguracion</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
